components/Table/Table.js

```diff
@@ -262,15 +248,27 @@ export class Table {
       if (this.callbacksMap.paginationCallback) {
         pageNumberBtn.addEventListener('click', () => {
           const currentPage = i;
-          this.callbacksMap.paginationCallback(
-            currentPage,
-            this.amountElOnPage
-          );
+          this.callbacksMap.paginationCallback(currentPage, this.amountElOnPage);
         });
       }
     }
   }
 
+  createModalWindow(desiredObject, rowId) {
+    const modalWindow = new Modal(this.tableContainer);
+    modalWindow.renderModal(desiredObject, rowId);
+  }
+
+  callModal() {
```

Comment:
As i see, you're adding the listener here
If I see the method, that called `callModal`, i'd think that when I call it - it will render the Modal but not that it'll add the listener

Reply:
And additionally, this listener relate to the table more than to the modal I guess